src/components/sidebar/Sidebar.jsx

```diff
@@ -0,0 +1,86 @@
+import React, { useState } from "react";
+import { SidebarData } from "../../data/Data";
+import { NavLink } from "react-router-dom";
+import {
+  RiHandCoinLine,
+  RiInformationLine,
+  RiMenuFoldFill,
+} from "react-icons/ri";
+
+import { motion } from "framer-motion";
+
+import "../../components/sidebar/Sidebar.css";
+
+const Sidebar = () => {
+  const [expanded, setExpaned] = useState(false);
+
+  const sidebarVariants = {
+    open : {
+      left : 0
+    },
+    closed : {
+      left : '-100%'
+    }
+  };
+
+  return (
+    <>
+      <div
+        className="menu-bars"
+        style={expanded ? {left: '30%'} : {left: '2%'}}
+        onClick={() => setExpaned(!expanded)}
+      >
+        <RiMenuFoldFill />
+      </div>
+
+      <motion.div
+        className="sidebar"
+        variants={sidebarVariants}
+        animate={expanded ? 'open' : 'closed'}
+      >
+        <div>
+        <div className="logo">
+          <RiHandCoinLine />
+          <span className="logo-text">
+            Crypto<span>Trade</span>
+          </span>
+        </div>
+
+        <div className="nav-menu">
+          <div className="links">
+            {SidebarData.map((item, index) => (
```

Comment:
destructure item here

Reply:
destructuring works for an only the title not for the icon